libraries/chain/include/graphene/chain/database.hpp

```diff
@@ -410,6 +410,16 @@ namespace graphene { namespace chain {
           */
          ///@{
          int match( const limit_order_object& taker, const limit_order_object& maker, const price& trade_price );
+         /***
+          * @brief Match the two orders
+          * @param taker the taker
```

Comment:
`* @ param maker, puts a limit order on the order book`
`* @ param taker, fills a maker's limit order removing it from the order book`

Reply:
Fixed, changed your suggested wording a bit for clarity and accuracy.